api/src/main/resources/application.conf

```diff
@@ -4,6 +4,7 @@ akka {
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
   http.parsing.max-content-length = 1G
   http.server.default-host-header = "cfpb.gov"
+  test.timefactor=30.0
```

Comment:
Does it need to be this high? This means that `expectMsg` assertions will take this value times `akka.test.single-expect-default` (3 seconds by default)